src/src.pro

```diff
@@ -18,7 +18,7 @@
 #    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
```

Comment:
I had a comment written out about this but on second thoughts I _thunked_(?) that this would work for the moment and cancelled the comment.

However, after I got like the constipated mathematician {_he sat down and worked it out with a pencil_} I realised the logic WAS wrong after all...